cuda_pathfinder/cuda/pathfinder/_dynamic_libs/load_nvidia_dynamic_lib.py

```diff
@@ -77,22 +77,20 @@ def load_nvidia_dynamic_lib(libname: str) -> LoadedDL:
            - Scan installed distributions (``site-packages``) to find libraries
              shipped in NVIDIA wheels.
 
-        2. **OS default mechanisms / Conda environments**
+        2. **Conda environment**
+
+           - Conda installations are discovered via ``CONDA_PREFIX``, which is
+             predefined in activated conda environments (see
```

Comment:
```suggestion
             defined in activated conda environments (see
```

I don't think the `pre` prefix is making a useful distinction beyond just "defined".

Reply:
Changed to "defined automatically": commit bd5e2ae58d69c3058542d2648ccbc933ec3d37d5

This is for people not-so-familiar with Conda. That one extra word is to reassure them that the don't have to think about defining the variable themselves.